path-traversal/COP4331-Large-Project_FoodGram/api.js

```
@@ -0,0 +1,611 @@
+
+  try{
+    // Check on if id is valid
+    if(!mongoose.Types.ObjectId.isValid(req.query.token)) {
+      var ret = {id: -1, error: "User not found"}
+      return res.json(ret);
+    }
+
+    var user = await User.findOne({_id: req.query.token})
+    if (!user)
+    {
+      var ret = {id: -1, error: "User not found"}
```

Biome (2.1.2)

[error] Shouldn't redeclare 'ret'. Consider to delete it or rename it.

'ret' is defined here:


(lint/suspicious/noRedeclare)